ext/Errno/Errno_pm.PL

```diff
@@ -272,6 +274,9 @@ sub write_errno_pm {
 
     # Write Errno.pm
 
+    open OUT, '>', $errno_tmp or die "Cannot open $errno_tmp: $!";
+    select OUT;
```

Comment:
Instead of this select, can't we just `print $out <<"EDQ"`

Reply:
I tried this approach.  The problem is that the generated file is not nicely contained within one single heredoc.  There are 9 separate `print` statements that are used to generate `Error.pm`.
```
280:    print <<"EDQ";
297:        print <<"CONFIG_CHECK_END";
306:    print <<"EDQ";
320:	print "\t$err => $err{$err},\n";
323:print <<'ESQ';
368:    print "\t",$k,"\n    )],\n";
371:	print "    WINSOCK => [qw(\n";
374:	print "\t",$k,"\n    )],\n";
377:    print ");\n\n";
379:    print <<'ESQ';
```
The `select` enables us to avoid typing `print $OUT` in each of those statements.  I suspect that was Gurusamy Sarathy's reasoning when he introduced the `select` back in eab60bb1f2e in 1998.